torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1234,12 +1242,29 @@ def _local_state_dict(self, *args: Any, **kwargs: Any) -> Any:
         with self.state_dict_type(self, StateDictType.LOCAL_STATE_DICT):
             return self.state_dict(*args, **kwargs)
 
+    def _full_post_load_state_dict_hook(self) -> None:
+        # We should exit summon_full_params context.
+        assert getattr(self, '_entered_full_param_ctx', None) is not None
```

Comment:
Asserts are usually not actionable by users when they lack a message.

Reply:
Fixing in next version